client/my-sites/plans/controller.jsx

```diff
@@ -29,7 +29,7 @@ export function plans( context, next ) {
 			return page.redirect( `/plans/${ context.params.site }` );
 		}
 		setJetpackPlansHeader( context );
-		return productSelect( '/plans/:site' )( context, next );
```

Comment:
I don't understand the change here, since it's not limited by the offer flow experiment. Can you shed light on that?

Reply:
`productSelect` expects a parameter called `rootUrl`. Since now all paths have the site's slug at the end, we need to set `rootUrl` to be just `/plans` and not `/plans/:site`. 

Also, everything inside this if statement is limited to the Offer Reset experiment.
```js
	if ( showJetpackPlans( context ) ) {
		if ( context.params.intervalType ) {
			return page.redirect( `/plans/${ context.params.site }` );
		}
		setJetpackPlansHeader( context );
		return productSelect( '/plans' )( context, next );
	}
```
Because `showJetpackPlans` returns true only if `shouldShowOfferResetFlow()` returns true and if the site is not a WPCOM site. In other words, the new Plans/Selector page is only shown if the Offer Reset experiment is active and the site is a self-hosted Jetpack site. If any of those conditions is not true, we rely on the current Plans page (which handles Jetpack and WordPress.com plans).

Please let me know if this answers your question, or if you think I'm missing something.

Reply:
Got it, thanks!